### PR TITLE
add hibernate bom

### PR DESCRIPTION
**Pull request by stliu**

### CODE REVIEW

File: jboss-javaee-6.0-with-hibernate/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+This BOM builds on the Java EE full profile BOM, adding Hibernate family projects (Hibernate ORM, Hibernate Search,
```
Comment (pmuir) [inline jboss-javaee-6.0-with-hibernate/README.md]: "...adding the Hibernate family of projects" ... <- missing some articles and prepositions.

File: pom.xml [hunk 2]
```diff
@@ -37,6 +37,14 @@
         <version.org.jboss.as.arquillian.container>7.1.0.CR1b</version.org.jboss.as.arquillian.container>
         <version.org.jboss.jbossts.jbossjts>4.16.1.Final</version.org.jboss.jbossts.jbossjts>
 
+        <!-- Version of Hibernate projects -->
+        <version.org.hibernate>4.1.0.Final</version.org.hibernate>
+        <version.org.hibernate.validator>4.2.0.Final</version.org.hibernate.validator>
+        <version.org.hibernate.search>4.1.0.Beta2</version.org.hibernate.search>
+        <version.org.hibernate.common>4.0.1.Final</version.org.hibernate.common>
+        <version.org.hibernate.javax.persistence>1.0.1.Final</version.org.hibernate.javax.persistence>
```
Comment (pmuir) [inline pom.xml:45]: Shouldn't this be JPA2?

Comment (stliu) [inline pom.xml:45]: it is :D

File: jboss-javaee-6.0-with-hibernate/README.md (hunk 1 above)
Comment (stliu) [inline jboss-javaee-6.0-with-hibernate/README.md]: see new commit, is it better?

**Comment by stliu** [comment]
could someone take a look of this?

**Comment by pmuir** [comment]
This was merged and released

**Pull request closed without merging**